Update URLs to point at JuliaVTK org where applicable (#24)
JuliaVTK/ReadVTK.jl: README.md

# ReadVTK.jl

[![Docs-stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://juliavtk.github.io/ReadVTK.jl/stable)
[![Docs-dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://juliavtk.github.io/ReadVTK.jl/dev)
[![Build Status](https://github.com/JuliaVTK/ReadVTK.jl/workflows/CI/badge.svg)](https://github.com/JuliaVTK/ReadVTK.jl/actions?query=workflow%3ACI)
[![Coveralls](https://coveralls.io/repos/github/JuliaVTK/ReadVTK.jl/badge.svg?branch=main)](https://coveralls.io/github/JuliaVTK/ReadVTK.jl?branch=main)
[![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
<!-- [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5221552.svg)](https://doi.org/10.5281/zenodo.5221552) -->

With **ReadVTK.jl** you can read in data from VTK XML files in Julia. It aims to complement
the excellent package [WriteVTK.jl](https://github.com/JuliaVTK/WriteVTK.jl).

> ReadVTK is part of the [Trixi Framework](https://JuliaVTK.github.io).

Reviewer: ```suggestion
ReadVTK is part of the [JuliaVTK Framework](https://github.com/JuliaVTK).
```
We don't have a dedicated website (yet).

*Note: ReadVTK was mainly motivated by wanting to write proper tests for

> [Trixi2Vtk.jl](https://github.com/JuliaVTK/Trixi2Vtk.jl).

Reviewer: ```suggestion
[Trixi2Vtk.jl](https://github.com/trixi-framework/Trixi2Vtk.jl).
```

## Authors
ReadVTK is maintained by the

> [Trixi authors](https://github.com/JuliaVTK/Trixi.jl/blob/main/AUTHORS.md).

Reviewer: ```suggestion
[Trixi authors](https://github.com/trixi-framework/Trixi.jl/blob/main/AUTHORS.md).
```

Since ReadVTK is an open-source project, we are very happy to accept contributions
from the community. Please refer to [CONTRIBUTING.md](CONTRIBUTING.md) for more details. To get in
touch with the developers, join us on Trixi's

> [Slack workspace](https://join.slack.com/t/JuliaVTK/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)

Reviewer: ```suggestion
[Slack workspace](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
```
